src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,1291 @@
+      className={`w-full border-4 p-6 relative transition-all duration-500 ${signed ? 'border-(--toxic-green) bg-(--toxic-green)/5' : 'border-(--toxic-green)/30 bg-[color:var(--void-black)]'}`}
+    >
+      <div className='absolute top-0 right-0 p-2 border-l border-b border-(--toxic-green)/30 text-[8px] opacity-50'>
+        {t('ui:contract.confidential')}
+      </div>
+
+      <h3 className='text-xl font-bold tracking-[0.2em] uppercase mb-4 border-b-2 border-(--toxic-green)/30 pb-2'>
+        {t('ui:contract.title')}
+      </h3>
+
+      <div className='text-xs leading-relaxed opacity-80 flex flex-col gap-3 font-mono'>
+        <p dangerouslySetInnerHTML={{ __html: t('ui:contract.p1') }}></p>
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)